auth/api/http/keys/transport.go

```diff
@@ -55,7 +58,12 @@ func decodeIssue(_ context.Context, r *http.Request) (interface{}, error) {
 		return nil, apiutil.ErrUnsupportedContentType
 	}
 
-	req := issueKeyReq{token: apiutil.ExtractBearerToken(r)}
+	token := apiutil.ExtractBearerToken(r)
+	if strings.HasPrefix(token, patPrefix) {
+		return nil, apiutil.ErrUnsupportedTokenType
+	}
```

Comment:
This check is not required, Because in the above line 
`apiutil.ExtractBearerToken` have check like this `strings.HasPrefix(token, BearerPrefix)`
This will return empty string if token doesn't start with `Bearer`

Reply:
I think, this will be handled by service layer, Please check and remove if not needed